schemas/trademark-schema.json

```diff
@@ -0,0 +1,206 @@
+          "type": "string"
+        },
+        "mark_forms": {
+          "description": "Forms the mark takes",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "text",
+              "image",
+              "sensory_mark",
+              "three_d_mark"
+            ]
+          }
+        },
+        "mark_type": {
+          "description": "Free text description of type mark (as provided by register)",
+          "type": "string"
+        },
+        "mark_image": {
+          "properties": {
+            "uid": {
+              "description": "uid of image",
+              "type": "string"
+            },
+            "format": {
+              "description": "Type of image format",
+              "type": "string"
+            },
+            "url": {
+              "description": "URL of image",
+              "type": "string"
+            },
+            "description": {
+              "description": "Description of image in words",
+              "type": "string"
+            }
+          }
+        },
+        "classification": {
+          "type": "array",
+          "description": "Classification of the trademark",
+          "items": {
+            "oneOf": [{
+                "type": [
+                  "string"
+                ],
+                "minLength": 1
+              },
+              {
+                "$ref": "#/definitions/nice_classification"
+              },
+              {
+                "$ref": "includes/classification.json"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "uid": {
+      "description": "Unique id/reference for the trademark registration",
+      "type": "string"
+    },
+    "source_url": {
+      "description": "Url of the source of the data (e.g. download URL), or if there is not persistent URL the page from which it can be found (e.g. search page). This field is required.",
+      "type": "string",
+      "minLength": 11
+    },
+    "confidence": {
+      "description": "Confidence in accuracy of data",
+      "type": "string",
+      "enum": [
+        "HIGH",
+        "MEDIUM",
+        "LOW"
+      ]
+    },
+    "registration_date": {
+      "description": "Date on which the mark was registered",
+      "type": "string",
+      "format": "date"
+    },
+    "expiration_date": {
+      "description": "Date on which the registration ended, or is scheduled to expire",
+      "type": "string",
+      "format": "date"
+    },
+    "start_date": {
+      "description": "This is copied from the registration_date field",
+      "type": "string",
+      "format": "date"
+    },
+    "end_date": {
+      "description": "This is copied from the expiration_date field",
+      "type": "string",
+      "format": "date"
+    },
+    "retrieved_at": {
+      "description": "Date-time this was retrieved from the source",
+      "type": "string",
+      "format": "date-time"
+    },
+    "sample_date": {
```

Comment:
We think we can remove this and make start/end dates required (and also add date types for both of those). @CountCulture to review USPTO and GBIPO data to confirm.